packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -961,16 +961,14 @@ ensureIndex(Posts,
   { name: "posts.events" }
 );
 
-Posts.addView("pastEvents", (terms: PostsViewTerms) => {
+Posts.addView("eventsInTimeRange", (terms: PostsViewTerms) => {
   return {
     selector: {
       isEvent: true,
       groupId: terms.groupId ? terms.groupId : null,
-      baseScore: {$gte: 1},
     },
     options: {
       sort: {
-        baseScore: -1,
```

Comment:
Most of this PR was done pairing with Jim, and we decided that filtering/sorting events by `baseScore` wasn't useful.